scripts/cmd/build.js

```diff
@@ -111,17 +111,6 @@ export default async function build(...args) {
 		plugins: [
 			rebuildPlugin,
 			svelte({ isDev }),
-			...(copyWASM
-				? [
-						copy({
-							resolveFrom: 'cwd',
-							assets: {
-								from: ['./src/assets/services/vendor/squoosh/**/*.wasm'],
-								to: ['./dist/assets/services/vendor/squoosh'],
-							},
-						}),
-					]
-				: []),
```

Comment:
FYI I'm also changing this in main at https://github.com/withastro/astro/pull/11769/files#diff-3295a1ee2ce5eaeef758fac7b3eea7c5413dc7ccc11b42afcfbd25f0f384d753R68 that moves this for the build too, but looks like after this we can completely remove it, the `--copy-wasm` command, and `esbuild-plugin-copy`.

Letting you know in case you hit merge conflicts